src/Components/Publishing/News/NewsHeadline.tsx

```diff
@@ -34,9 +34,8 @@ const NewsHeadlineContainer = styled.div`
   text-align: left;
   max-width: 780px;
   width: 100%;
-  margin: 40px auto;
+  margin: 10px auto 30px auto;
```

Comment:
Could this be shortened to `10px auto 30px` since the side margins are the same?

Reply:
Sweet! I wasn't sure if I was right about that 😂